app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
The places in the code which currently reference Division.root rely on it existing.  So it feels more appropriate that the validation of existence is at the model level, not the controller level.  If the need for a root division disappears as the project evolves, then that will certainly take care of things, but it's currently depended upon, and in my mind conceptually part of the current Division contract.

If you really believe that an assert doesn't belong in the model class, I'll remove it, but it'll just result in less useful errors when debugging test code in situations where the data wasn't properly initialized.